Formula/c/catgirl.rb

```diff
@@ -0,0 +1,28 @@
+class Catgirl < Formula
+  desc "Terminal IRC client"
+  homepage "https://git.causal.agency/catgirl/about/"
+  url "https://git.causal.agency/catgirl/snapshot/catgirl-2.2a.tar.gz"
+  sha256 "c6d760aaee134e052586def7a9103543f7281fde6531fbcb41086470794297c2"
+  license "GPL-3.0-or-later"
+
+  depends_on "ctags" => :build
+  depends_on "pkgconf" => :build
+
+  depends_on "libretls"
+  uses_from_macos "ncurses"
+
+  def install
+    args = %W[
+      --mandir=#{man}
+    ]
+
+    system "./configure", "--disable-silent-rules", *std_configure_args, *args
+    system "make"
+    system "make", "install"
+  end
+
+  test do
+    stdout, = PTY.spawn(bin/"catgirl")
+    assert_match "catgirl: host required", stdout.readline
```

Comment:
```suggestion
    stdout = shell_output(bin/"catgirl")
    assert_match "catgirl: host required", stdout
```

Reply:
@SMillerDev I think that broke the build - maybe because the expected message is supposed to come from stderr?